elliott/elliottlib/util.py

```
@@ -734,3 +736,245 @@ def get_advisory_boilerplate(runtime, et_data, art_advisory_key, errata_type):
+                    artifact_type == 'application/vnd.konflux-ci.attached-artifact'
+                    and 'related-images' in attached_media_type
+                ):
+                    digest = referrer.get('digest')
+                    logger.info(f"Found related-images artifact with digest: {digest}")
+                    break
+
+            if not digest:
+                available_types = [r.get('artifactType') for r in referrers]
+                raise RuntimeError(
+                    f"No attached artifact found with expected type 'application/vnd.konflux-ci.attached-artifact'. Available types: {available_types}"
+                )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)